tests/bluetooth/tester/src/btp/btp_aics.h

```diff
@@ -0,0 +1,26 @@
+/* aics.h - Bluetooth tester headers */
+
+/*
+ * Copyright (c) 2022 Codecoup
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <stdint.h>
+#include <zephyr/sys/util.h>
+
+#define AICS_READ_SUPPORTED_COMMANDS	0x01
+
+#define AICS_INIT			0x02
+#define AICS_SET_GAIN			0x03
+#define AICS_MUTE			0x04
+#define AICS_UNMUTE			0005
+#define AICS_MAN_GAIN			0x06
+#define AICS_AUTO_GAIN			0x07
+#define AICS_MAN_GAIN_ONLY		0x08
+#define AICS_AUTO_GAIN_ONLY		0x09
+#define AICS_DESCRIPTION		0x10
```

Comment:
Are these defined by some specification? If so, please add a reference. 

Should these be put into an `enum` as per the Zephyr code guidelines? (this comment applies to all "blocks" of `#define`s)

Reply:
This could be done in another PR, as this will also require a change in other already implemented services.

Reply:
> This could be done in another PR, as this will also require a change in other already implemented services.

Only if we don't want to have 2 different styles. Personally I don't see any issue with using `enum` here, when it's not done in other files, but I do find it weird not to do it "properly" when adding a new file which is the case here. 

@sjanc @MariuszSkamra  What do you think?

Reply:
I would keep it consistent and keep using `#define`

Reply:
yes, lets keep it as is in this PR,   I'll be working later this year on some cleanups in tester app as we continue adding le audio support.  Will handle those there.